metadata/client.go

```diff
@@ -220,6 +220,7 @@ type FeatureDef struct {
 	Location    interface{}
 	Tags        Tags
 	Properties  Properties
+	Category    FeatureVariantCategory
```

Comment:
Not in love with `Category` as a name

Reply:
Yeah, `type` was already taken, so I tried to figure out an acceptable synonym. `VariantType`?